futtat.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cat input.txt | ./a.out > output.txt
```

Comment:
miert nem csak siman `./a.out < input.txt > output.txt`?

Reply:
Így is teljes értékű a parancs és szerintem átláthatóbb. Performancia beni különbség nincs a kettő között